### PR TITLE
Update persisted queries doc

### PR DESCRIPTION
**Pull request by martinbonnin**

### CODE REVIEW

File: docs/source/advanced/persisted-queries.mdx [hunk 1]
```diff
@@ -1,56 +1,143 @@
...
+> **Note:** You can use _either_ Apollo Server _or_ Apollo Router for APQs. They don't need to be used together.
+
+Persisted queries is currently in [preview](/resources/product-launch-stages#preview) and has the following requirements:
+- Apollo Kotlin (v3.8.2+)
+- [Apollo Router](/router) (v1.25.0+)
+- [GraphOS Enterprise plan](/graphos/enterprise/)
+
+## Implementation steps
+
+Both persisted queries and APQs require you to configure your client makes requests. If you intend to use persisted queries for safelisting, you also need to generate the operations manifest.
```
Comment (BoD) [inline docs/source/advanced/persisted-queries.mdx:38]:
```suggestion
Both persisted queries and APQs require you to configure how your client makes requests. If you intend to use persisted queries for safelisting, you also need to generate the operations manifest.
```

File: docs/source/advanced/persisted-queries.mdx [hunk 2]
```diff
@@ -1,56 +1,143 @@
...
+Ensure your Rover CLI version is `0.17.2` or later. Previous versions of Rover don't support publishing operations to a PQL.
+[Download the latest version.](/rover/getting-started/)
+
+</blockquote>
+
+If you want to preregister your manifest as a safelist of trusted operations, you need to publish it to a PQL using the [Rover CLI](/rover/):
+
+```bash title="Example command"
+rover persisted-queries publish my-graph@my-variant \
+  --manifest ./persistedQueryManifest.json
```
Comment (BoD) [inline docs/source/advanced/persisted-queries.mdx:104]: Nitpick, may help clarity to use a realistic path
```suggestion
  --manifest app/build/generated/manifest/apollo/api/persistedQueryManifest.json
```

**Review by BoD — APPROVED** [review]
👍

**Review by Meschreiber — APPROVED** [review]
TY for this @martinbonnin ! I'm going to try to make some shared components before launch on Thursday. Alright if I commit directly to this branch with those components if I get around to it?

**Comment by Meschreiber** [comment]
Ah, also--could we move this to the `/fetching` section in the nav so it has the same placement as iOS? I agree this is an advanced feature, but I'd love to "promote" it higher up in the nav under fetching.

If you're ok with that @martinbonnin , I can commit that change + add a redirect.

**Comment by martinbonnin** [comment]
@Meschreiber please do, feel free to push any change here.

**Comment by Meschreiber** [comment]
For some reason these changes aren't being deployed 🤔. Going to see if reverting and remerging fixes it.

**Pull request merged**